IdentityCore/src/cache/MSIDMacTokenCache.m

```diff
@@ -172,7 +172,10 @@ - (NSMutableDictionary *)cache
 
```

Comment:
seems function `- (NSMutableDictionary *)cache` is no longer used anywhere :)

Reply:
It's used in self.cache. It overrides the accessor.